plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -249,12 +249,12 @@ public static function get_json_schema(): array {
 						'width'  => array(
 							'type'     => 'integer',
 							'required' => true,
-							'minimum'  => 0,
+							'minimum'  => 1,
 						),
 						'height' => array(
 							'type'     => 'integer',
 							'required' => true,
-							'minimum'  => 0,
+							'minimum'  => 1,
```

Comment:
Important: Note how the minimum width/height for a URL Metric is now 1px (i.e. `positive-int`)